spec/02-integration/04-admin_api/06-certificates_routes_spec.lua

```diff
@@ -893,8 +893,15 @@ describe("Admin API: #" .. strategy, function()
 
         -- make sure we did not add any certificate or sni
         local json = get_certificates()
-        json.updated_at = nil
-        json_before.updated_at = nil
+
+        -- We don't want to compare the updated_at timestamp, so remove it from the result
+        for _, cert in ipairs(json.data) do
+          cert.updated_at = nil
+        end
+        for _, cert in ipairs(json_before.data) do
+          cert.updated_at = nil
+        end
+
```

Comment:
Why not put this directly inside `get_certificates`?

Reply:
Good suggestion! Changed